src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
its fields not constants, so they should not be static
```suggestion
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```

Reply:
not fixed